_layouts/base.html

```diff
@@ -160,8 +160,8 @@ <h1><a href="{{ site.baseurl }}/">{{ site.title }}</a></h1>
         }
       }
     </script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.2.0/anchor.min.js"
-            integrity="sha256-m1eTvwEHwmsw4+XKF7BshClVJEPwTVycveNl0CS0Mkk="
+    <script src="https://cdn.jsdelivr.net/npm/anchor-js@4.3.1/anchor.min.js"
+            integrity="sha256-0WMZ9PF4b2hTF66Eglv/9H5Vwk6lnOG4AbmCJxo96WQ="
```

Comment:
```suggestion
    <script src="https://cdn.jsdelivr.net/npm/anchor-js@5.0.0/anchor.min.js"
            integrity="sha256-aQmOEF2ZD4NM/xt4hthzREIo/2PFkOX/g01WjxEV7Ys="
```

newer version out now